GOVERNANCE-maintainer.md

```diff
@@ -0,0 +1,106 @@
+# Project Governance
+
+This is an example governance document for projects that use the very common
+"maintainer council" system of governance.  See [GOVERNANCE.md](/GOVERNANCE.md)
+for more information.  Thanks to the Jaeger Project for portions of the text
+below.
+
+This is a template document for CNCF projects that requires editing
+before it is ready to use. Read the markdown comments, `<!-- COMMENT -->`, for
+additional guidance. The raw markdown uses `TODO` to identify areas that
+require customization.  Replace PROJECTNAME with the name of your project.
+
+- [Maintainers](#maintainers)
+- [Becoming a Maintainer](#becoming-a-maintainer)
+- [Meetings](#meetings)
+- [CNCF Resources](#cncf-resources)
+- [Code of Conduct Enforcement](#code-of-conduct)
+- [Voting](#voting)
+
+## Maintainers
+
+PROJECTNAME Maintainers have write access to the [project GitHub repository](TODO).
+They can merge their own patches or patches from others. The current maintainers
+can be found in [OWNERS](./OWNERS).  Maintainers collectively manage the project's
+resources and contributors.
+
+This privilege is granted with some expectation of responsibility: maintainers
+are people who care about the PROJECTNAME project and want to help it grow and
+improve. A maintainer is not just someone who can make changes, but someone who
+has demonstrated his or her ability to collaborate with the team, get the most
+knowledgeable people to review code and docs, contribute high-quality code, and
+follow through to fix issues (in code or tests).
+
+A maintainer is a contributor to the project's success and a citizen helping
+the project succeed.
+
+## Becoming a Maintainer
+
+<!-- If you have full Contributor Ladder documentation that covers becoming
+a Maintainer or Owner, then this section should instead be a reference to that
+documentation -->
+
+To become a Maintainer you need to demonstrate the following:
```

Comment:
do we want to say something about security somewhere?

Reply:
do you have example text?